src/components/ShowMore/index.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Text, View} from 'react-native';
+import _ from 'underscore';
+import Button from '@components/Button';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useLocalize from '@hooks/useLocalize';
+import * as NumberFormatUtils from '@libs/NumberFormatUtils';
+import stylePropTypes from '@styles/stylePropTypes';
+import styles from '@styles/styles';
+import themeColors from '@styles/themes/default';
+
+const propTypes = {
+    /** Additional styles for container */
+    containerStyle: stylePropTypes,
+
+    /** A number of current showed items */
+    currentCount: PropTypes.number,
+
+    /** A number of total items */
+    totalCount: PropTypes.number,
+
+    /** A handler fires when button has been pressed */
+    onPress: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+    containerStyle: {},
+    currentCount: undefined,
+    totalCount: undefined,
+};
+
+function ShowMore({containerStyle, currentCount, totalCount, onPress}) {
```

Comment:
I recommend we call this `ShowMoreButton` since it's a button component.

Reply:
Done - b4d1537.